db/dcrpg/internal/txstmts.go

```diff
@@ -66,11 +66,26 @@ const (
 
 	SelectTxIDHeightByHash = `SELECT id, block_height FROM transactions WHERE tx_hash = $1;`
 
+	SelectTxsPerDay = `SELECT to_timestamp(time)::date as date, count(*) FROM transactions GROUP BY date ORDER BY date;`
+
 	SelectFullTxByHash = `SELECT id, block_hash, block_height, block_time, 
 		time, tx_type, version, tree, tx_hash, block_index, lock_time, expiry, 
 		size, spent, sent, fees, num_vin, vin_db_ids, num_vout, vout_db_ids 
 		FROM transactions WHERE tx_hash = $1;`
 
+	SelectTicketsOutputCountByAllBlocks = `SELECT block_height,
+		SUM(CASE WHEN num_vout = 3 THEN 1 ELSE 0 END) as solo,
+		SUM(CASE WHEN num_vout = 5 THEN 1 ELSE 0 END) as pooled
```

Comment:
These cases are not generally correct for identifying solo and pool tickets as either may be created with a large number of outputs, if they are created with the appropriate number of inputs.  3 and 5 are the most common number of outputs for solo and pool tickets, however.

The way to identify pooled tickets would be to identify addresses with P2SH prefixes (pool) that decode to M-of-N multi-sig scripts.

Reply:
Can this be easily done on the transactions table or I will have to write additional queries?

Reply:
Unfortunately it can't be done reliably by looking at the #vins and #vouts, and there's nothing else in the transactions table that can help.

The way to check would be to look at pkScript of the first output.  If it's script hash, then it's pool.  In the `vouts` table, there is `script_type` column.  It should be a string equal to one of these:

https://github.com/decred/dcrd/blob/eec4f96b0d890905bd871d02be498618e1aec745/txscript/standard.go#L49-L64

```go
// scriptClassToName houses the human-readable strings which describe each
// script class.
var scriptClassToName = []string{
	NonStandardTy:     "nonstandard",
	PubKeyTy:          "pubkey",
	PubkeyAltTy:       "pubkeyalt",
	PubKeyHashTy:      "pubkeyhash",
	PubkeyHashAltTy:   "pubkeyhashalt",
	ScriptHashTy:      "scripthash",
	MultiSigTy:        "multisig",
	NullDataTy:        "nulldata",
	StakeSubmissionTy: "stakesubmission",
	StakeGenTy:        "stakegen",
	StakeRevocationTy: "stakerevoke",
	StakeSubChangeTy:  "sstxchange",
}
```

I suggest declaring a ticket a pooled ticket if the type is `"scripthash"` or solo in any other case.

Reply:
ok. Noted